### PR TITLE
Support `Unload` statement

### PR DESCRIPTION
**Pull request by jonathanlehto**
Adding UNLOAD as a recognized statement

Redshift -- https://docs.aws.amazon.com/redshift/latest/dg/r_UNLOAD.html
Athena -- https://docs.aws.amazon.com/athena/latest/ug/unload.html

### CODE REVIEW

File: tests/sqlparser_common.rs [hunk 1]
```diff
@@ -8351,6 +8351,67 @@ fn parse_binary_operators_without_whitespace() {
     );
 }
 
+#[test]
+fn parse_unload() {
+    let unload = verified_stmt("UNLOAD(SELECT cola FROM tab) TO 's3://...' WITH (format = 'AVRO')");
```
Comment (alamb) [inline tests/sqlparser_common.rs:8356]: fascinating -- I haven't seen `UNLOAD` before

**Review by alamb — APPROVED** [review]

**Comment by alamb** [comment]
Thank you very much @jonathanlehto  for this contribution

**Comment by alamb** [comment]
I took the liberty of merging the branch up from main to fix a merge conflict, as well as fixing a clippy issue

**Comment by alamb** [comment]
Thanks again @jonathanlehto

**Comment by jonathanlehto** [comment]
No problem, thank you!

**Pull request merged**